test/cql-pytest/test_bloom_filter.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# Test inserts `N` rows into table, flushes it 
+# and tries to read `M` non-existing keys.
+# Then bloom filter's false-positive ratio is checked.
+
+import pytest
+import random
+import nodetool
+from util import new_test_table
+
+def test_bloom_filter(cql, test_keyspace):
+    N = 10000 # Number of writes
+    M = 5000  # Number of reads
+    fp_chance = 0.1
+
+    keys = random.sample(range(2147483647), N+M)
```

Comment:
Is using random keys an important part of this test? What happens if we just take the keys 1...N and that's it? Will we expect different results?

If it's not random, you'll have the same results on every run and not have a chance of the test failing once in a while by random.

Reply:
Non random keys will always return the same result. Imo, using random keys it's easier to pick if sth is wrong with bloom filter.

Reply:
I think that random keys will basically guarantee that at some probability the test will fail. Maybe this probability is infinitesimal and we have nothing to worry about, but I guess at a minimum you should run this test a thousand times (this is very easy to do as it's so fast) and see that it doesn't fail.

If the purpose of this test is to catch regressions in the Bloom Filter code and not to confirm its probabilistic features, I think it doesn't need random data.

Reply:
With 1000 runs, 4 tests failed. I changed test to non random keys.

Reply:
I don't see this change - it still seems random. What am I missing?